api/BeforeUnloadEvent.json

```diff
@@ -77,10 +77,10 @@
               "version_added": "44"
             },
             "safari": {
-              "version_added": null
+              "version_added": "11"
```

Comment:
This must have involved manual testing or research, can you update the description with what you did?

Reply:
This one was tested based upon the same test as #11002.  Since we have an active discussion there regarding what this means, I'll split this off into its own PR.